test/stream_data_test.exs

```diff
@@ -512,6 +512,21 @@ defmodule StreamDataTest do
     assert check_all(list_of(boolean()), options, property) == {:ok, %{}}
   end
 
+  test "check_all/3 with :rand.export_seed()" do
+    :rand.uniform()
```

Comment:
@josevalim is this to set a seed? If so, what would be the downside of being explicit and doing:

```elixir
seed = :rand.seed_s(:exs1024)
options = [initial_seed: :rand.export_seed_s(seed)
```

?

Reply:
No downside, that's better, yes!